#parameters to be used with gcar_sim.cpp

########################################
#  General settings(non reconfigurable)
debug_on: true;

########################################
# Topic name(Input)
#   (un-modifiable in reconfigure)
#   strtop_ctrl: 
strtop_ctrl: "/ctrl_si"

########################################
# Topic name(Output)
#   (un-modifiable in reconfigure)
#   strtop_odom: topic for nav_msgs::Odometry messages
strtop_odom: "/insekf/odom"

########################################
#  pose2d: [x(m) , y(m) , a(rad)]
pose2d: [113.0 , 176.0, 2.1]

